applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_isotropic_damage.cpp

```diff
@@ -118,7 +118,6 @@ void GenericSmallStrainIsotropicDamage<TConstLawIntegratorType>::CalculateMateri
 
             if (r_constitutive_law_options.Is(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR)) {
                 noalias(r_tangent_tensor) = (1.0 - damage) * r_constitutive_matrix;
-                TConstLawIntegratorType::YieldSurfaceType::CalculateEquivalentStress(auxiliar_integrated_stress_vector, r_strain_vector, uniaxial_stress, rValues);
```

Comment:
Can this be corrected in finite strain case?

Reply:
the damage is not in finite strain doesn't it?

Reply:
True